CourseApp/Program.cs

```diff
@@ -1,13 +1,75 @@
 ﻿namespace CourseApp
 {
     using System;
+    using System.Collections;
+    using static System.Math;
 
     public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine($"Hello world");
+            var a = 2.0;
+            var b = 3.0;
+            ArrayList listTaskOne = new ArrayList() { 0.11, 0.36, 0.05 };
```

Comment:
Please, do not use ArrayList, use List<T> instead
https://docs.microsoft.com/en-us/dotnet/api/system.collections.arraylist?view=net-5.0

Reply:
Fixed